### PR TITLE
Document the `kubectl` command builder

### LINKED ISSUE #748: Slack Common Commands

### PR DESCRIPTION
**Pull request by mszostok**
## Description

Changes proposed in this pull request:

- Document the `kubectl` command builder

Preview URL: https://kc-cmd-builder.botkube-docs.pages.dev/docs/next/usage/executing-kubectl-commands/#interactive-kubectl-commands-builder

## Related issue(s)

Fix https://github.com/kubeshop/botkube/issues/748

### CODE REVIEW

File: docs/usage/executing-commands.md [hunk 1]
```diff
@@ -27,15 +27,35 @@ This command needs to be executed from configured channel else use `--cluster-na
 ![get_pods](assets/get_namespaces.png)
 ![get_pods](assets/mm_get_ns.png)
 
-### Checking allowed commands
+## Interactive kubectl commands builder
+
+The interactive `kubectl` command builder allows you to construct a `kubectl` command just by selecting items from dropdowns. It is especially useful with the mobile application when typing the command is harder.
```
Comment (ezodude) [inline docs/usage/executing-commands.md:32]:
```suggestion
Use the interactive `kubectl` command builder to construct a `kubectl` command just by selecting items from dropdowns. This is especially useful on mobile when typing the command is harder.
```

File: docs/usage/executing-commands.md [hunk 2]
```diff
@@ -27,15 +27,35 @@ This command needs to be executed from configured channel else use `--cluster-na
...
+
+An additional advantage is the resource name dropdown that is pre-populated with all relevant resource names. It increases discoverability and gives you an option to select e.g. a Pod name without the need to type or copy-paste it.
```
Comment (ezodude) [inline docs/usage/executing-commands.md:34]:
```suggestion
The builder includes a resource name dropdown list. This is pre-populated with all the relevant resource names. It's great for discovering resources with the option to select them. E.g. Just grab a Pod name without needing to type or copy-and-paste.
```

**Review by ezodude — APPROVED** [review]
Looks good, added a couple of suggestions.

**Comment by mszostok** [comment]
Thanks for the review 🙇  I applied all suggestions 👍

**Pull request merged**